### DISCUSSION TITLE
Want to know the release date of 1.0.0

### DISCUSSION THREAD

**Discussion by zse4t7i** [thuliteio/doks#1073] [Q&A] [answered]
Hello, I'm trying to move my articles from hexo to hugo, then I found Doks and saw the release of 1.0.0.rc2. Can I know the specific time of the release date of 1.0.0? And will there be any major changes to the documentation?

**Comment by h-enk** [comment, marked as answer]
Since Doks v1.0 is completely restructured /now properly based on Hyas, updated documentation (including a migration guide) is key before shipping. I'm in the process of finishing the documentation for Hyas first, next is the documentation for Doks.

It's really hard to give a date since I'm busy with a lot of stuff (that needs prioirty). Know that I'm aiming to ship Doks v1.0 asap.

**Reply by zse4t7i** [reply]
Got it! Thanks for your work. I can wait it~ :wink:

**Comment by Yvand** [comment]
@h-enk I'm using your template on 2 project sites and I really enjoy it, so before all I wanted to say thank you.
I read your previous message, but it was 2+ months ago, are you now able to give some visibility on the publish date of v1.0 ?

I'm eagerly waiting for it to do major changes on my sites, and I very much wish to do them with the v1.0 , but it depends if it comes soon or not, as I cannot postpone it for too long.

Many thanks again for your great work

**Reply by h-enk** [reply]
Hey @Yvand, thanks for your compliments — and your patience. 

I will be releasing Doks 1.0 — whatsoever — at the end of next week at the latest.

**Reply by nickpoorman** [reply]
Looking forward to it!

**Comment by h-enk** [comment]
Just shipped  https://github.com/h-enk/doks/releases/tag/v1.0.0

Updated documentation is in the making /will follow shortly!

**Reply by bigethan** [reply]
Congratulations and thank you!!!
